pyproject.toml

```diff
@@ -1,11 +1,15 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
 [project]
 name = "stactask"
 version = "0.6.1"
```

Comment:
With all the modernization efforts can I convince everyone that we should be using a dynamic version??? 😅🙏

Reply:
Yep.